sdk/build.gradle

```diff
@@ -0,0 +1,48 @@
+subprojects {
+    apply plugin: 'com.android.library'
+    apply plugin: 'kotlin-android'
+    apply plugin: 'kotlin-android-extensions'
+
+    def ext = rootProject.ext
+
+    android {
+        compileSdkVersion ext.androidCompileVersion
+        buildToolsVersion ext.buildToolsVersion
+
+        defaultConfig {
+            minSdkVersion ext.minSDKVersion
+            targetSdkVersion ext.targetSDKVersion
+            testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+            versionCode ext.versionCode
+            versionName ext.versionName
+        }
+
+        buildTypes {
+            release {
+                minifyEnabled false
+                proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            }
+        }
+    }
+
+    dependencies {
```

Comment:
Other than adding the dependency version variables nothing here has changed. Its just been moved from the core SDK build script to this level. All these dependencies are common to all submodules so it's high level stuff like setting up Kotlin, JUnit, Mockito, etc